cdap-ui/webpack.config.js

```diff
@@ -20,12 +20,12 @@ var path = require('path');
 var LodashModuleReplacementPlugin = require('lodash-webpack-plugin');
 var plugins =
 [
-  new webpack.DefinePlugin({
-    'process.env':{
-      'NODE_ENV': JSON.stringify("production"),
-      '__DEVTOOLS__': false
-    },
-  }),
+  // new webpack.DefinePlugin({
```

Comment:
This could be reverted back I guess. We can handle this based on the environment variables.